Automattic/wp-calypso
# Signup: Add a user testimonial to specific pages in signup

WIP, please do not merge. Continuation of work done in #27862 

#### Changes proposed in this Pull Request

* Adds a user testimonial to specific pages in signup.
* Testimonial becomes a new component included in the StepWrapper component.

#### Testing instructions

* Switch to this PR
* Check About and User login pages for testimonial at the bottom.
